diff --git a/3rdparty/assimp/assimp.cmake b/3rdparty/assimp/assimp.cmake
index e6da41c35..3995d7321 100644
--- a/3rdparty/assimp/assimp.cmake
+++ b/3rdparty/assimp/assimp.cmake
@@ -17,8 +17,8 @@ endif()
 ExternalProject_Add(
     ext_assimp
     PREFIX assimp
-    URL https://github.com/assimp/assimp/archive/cfed74516b46a7c2bdf19c1643c448363bd90ad7.tar.gz
-    URL_HASH SHA256=b2f1c9450609f3bf201aa63b0b16023073d0ebb1c6e9ae5a832441f1e43c634c
+    URL https://github.com/assimp/assimp/archive/0b19b7d73b8c6e40925114e4fcc936f48ee4323d.tar.gz
+    URL_HASH SHA256=ca1f08c38e63835d80ac709c94c7243fde50cb514f9c28b1723496a26217f8b8
     DOWNLOAD_DIR "${OPEN3D_THIRD_PARTY_DOWNLOAD_DIR}/assimp"
     UPDATE_COMMAND ""
     CMAKE_ARGS
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6e5784bb4..69500cfcd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,7 +110,7 @@ option(USE_SYSTEM_PNG             "Use system pre-installed png"             OFF
 option(USE_SYSTEM_PYBIND11        "Use system pre-installed pybind11"        OFF)
 option(USE_SYSTEM_QHULLCPP        "Use system pre-installed qhullcpp"        OFF)
 option(USE_SYSTEM_STDGPU          "Use system pre-installed stdgpu"          OFF)
-option(USE_SYSTEM_TBB             "Use system pre-installed TBB"             OFF)
+option(USE_SYSTEM_TBB             "Use system pre-installed TBB"             ON)
 option(USE_SYSTEM_TINYGLTF        "Use system pre-installed tinygltf"        OFF)
 option(USE_SYSTEM_TINYOBJLOADER   "Use system pre-installed tinyobjloader"   OFF)
 option(USE_SYSTEM_VTK             "Use system pre-installed VTK"             OFF)
